content/en/security_chaos_engineering.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Security Chaos Engineering
+status: Completed
+category: concept
+---
+
+## What it is
+Security Chaos Engineering or SCE is a discipline of performing proactive security experimentation on a distributed system in order to build confidence in the System's capability to withstand turbulent and malicious conditions. Security Chaos Engineers use scientific method loop (steady-state, hypothesis, continuous verification, lesson learned, implement mitigation). This approach helps SOC Engineers, [Site Reliability Engineers](/site_reliability_engineering/), [DevOps](/devops/) and [DevSecOps](/devsecops/) Engineers to create confidence on the system, increase cyber resiliency and improve Security [Observability](/observability/).
+
+## Problem it addresses
+An SRE or Cyber Security Engineer's main priority is to restore the service as fast as possible. Given their nature of incidents and practices, both deal with pre-failure and post-failure situations. However, they perceive as opposing disciplines. SREs follow ultimate reliability, and Cyber Security Engineers focus on the System's solid security protection and defense strategy. Unfortunately, most security issues are challenging to discover and quickly patch, impacting the business and the correct functionality of the application or the System. Those lacks are usually tricky to find out in the development phase.
+
+## How it helps
+Security Chaos Engineering is built around [observability](/observability/) and cyber resiliency practices, aiming to uncover the "unknown unknowns" and build confidence in the System, increase cyber resiliency and improve Security observability. Engineering teams will progressively work against missing understanding for security concerns within complex infrastructure, platforms, and distributed systems. The expected results of Security Chaos Engineering are improvements in the cyber resiliency of the whole product, uncovering hidden security issues, exposing the classical blind spots, and preparing the teams for critical edge cases.
```

Comment:
Some of this section belongs in the "what it is" portion. "How it helps" should be limited to describing what SCE does that is beneficial and addresses the problem described in the preceding section.